pkg/specgen/generate/kube/kube.go

```diff
@@ -106,17 +106,21 @@ func ToSpecGen(ctx context.Context, containerYAML v1.Container, iid string, newI
 
 	// TODO: We dont understand why specgen does not take of this, but
 	// integration tests clearly pointed out that it was required.
-	s.Command = []string{}
 	imageData, err := newImage.Inspect(ctx)
 	if err != nil {
 		return nil, err
 	}
 	s.WorkDir = "/"
+	// We will use "Docker field name" internally here to avoid confusion
```

Comment:
Docker field name?

Why do we have the "Docker" adgective here?

Reply:
it's the language used in the k8s docs

Reply:
Just cause they use bad language does not mean we have to. :^)

Reply:
updated :)